Dockerfile.slim

```
@@ -0,0 +1,49 @@
+#---------------------------------------------------------------------------------------------
+# See LICENSE in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+ARG PYTHON_VERSION="3.11"
+
+FROM python:${PYTHON_VERSION} as builder
+
+WORKDIR /wheels
+
+RUN pip install --upgrade pip \
+    && pip install wheel
+
+COPY . .
+
+RUN pip wheel .
+
+RUN mkdir /wheels/recommended_plugins
+
+COPY ./recommended-plugins.txt ./recommended-plugins.txt
+
+RUN pip wheel .[recommended-plugins] --wheel-dir=/wheels/recommended_plugins
+
+FROM python:${PYTHON_VERSION}-slim
+
+# See http://label-schema.org for metadata schema
+# TODO: Add `build-date` and `version`
+LABEL maintainer="ApeWorX" \
+      org.label-schema.schema-version="2.0" \
+      org.label-schema.name="ape" \
+      org.label-schema.description="Ape Ethereum Framework." \
+      org.label-schema.url="https://docs.apeworx.io/ape/stable/" \
+      org.label-schema.usage="https://docs.apeworx.io/ape/stable/userguides/quickstart.html#via-docker" \
+      org.label-schema.vcs-url="https://github.com/ApeWorX/ape" \
+      org.label-schema.docker.cmd="docker run --volume $HOME/.ape:/home/harambe/.ape --volume $HOME/.vvm:/home/harambe/.vvm --volume $HOME/.solcx:/home/harambe/.solcx --volume $PWD:/home/harambe/project --workdir /home/harambe/project apeworx/ape compile"
```

This is just a general guide? Think this can be shortened
```suggestion
      org.label-schema.docker.cmd="docker run --volume $PWD:/home/harambe/project --workdir /home/harambe/project apeworx/ape run my_script.py"
```

Or something like that since we are not bundling any compilers here